examples/tutorials/asr_inference_with_ctc_decoder_tutorial.py

```diff
@@ -286,6 +296,54 @@ def forward(self, emission: torch.Tensor) -> List[str]:
 #
 
 
+######################################################################
+# Timestep Alignments
+# -------------------
+# Recall that one of the components of the resulting Hypotheses is timesteps
+# corresponding to the token IDs.
+#
+
+
+timesteps = beam_search_result[0][0].timesteps
+predicted_tokens = beam_search_decoder.idxs_to_tokens(beam_search_result[0][0].tokens)
+
+print(predicted_tokens, len(predicted_tokens))
+print(timesteps, timesteps.shape[0])
+
+######################################################################
+# Below, we visualize the token timestep alignments relative to the original waveform.
+#
+
+
+import matplotlib.pyplot as plt
```

Comment:
Can we consolidate all the import statements at `Preparation`, so that it fails fast?